static/js/country_cards.js

```diff
@@ -183,6 +190,65 @@ const setSearch = (event) => {
     selectBtn.firstElementChild.innerText = 'Filter by Region';
 }
 
+const changeGridLayout = (event) => {
```

Comment:
Zie comment in https://github.com/rafainsunza/country-data/pull/2/files#r1668170900

Daarbij zou dit ook veranderd moeten worden. Je kan in principe er van uit gaan dat je waneer je toggled, dat je of 'meer' of 'minder' tiles toggled. op basis daarvan kan je de rest me css afhandelen.

Dus, klik je meer tiles, dan kan je om je tile wrapper een class/data attributen togglen, die je dan in je css gebruikt.
En hetzelfde voor minder tiles